native/acter/src/api/verification.rs

```diff
@@ -1066,7 +1070,7 @@ impl VerificationController {
                                 msg.set_content("short_authentication_string".to_string(), short_authentication_string.join(","));
                             }
                             StartMethod::ReciprocateV1(content) => {
-                                let secret = serde_json::to_string(&content.secret).unwrap();
+                                let secret = serde_json::to_string(&content.secret).expect("couldn't convert secret to string");
```

Comment:
considering that this is in a callback, we probably don't want to panic here but instead do log if this doesn't work...
```suggestion
                            
let secret = match serde_json::to_string(&content.secret) {
    Ok(e) => e;
    Err(e) => {
       error!("Parsing RecipV1 secret failed", error);
       return;
    }
};
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1245/commits/a183faf7571b2a83fa1b0d3e33d6a15d2fb4f03b